core/gdxsv/gdxsv.cpp

```diff
@@ -815,22 +815,34 @@ void Gdxsv::WritePatchDisk2() {
 
     // Dirty widescreen cheat
     if (config::WidescreenGameHacks.get()) {
+        bool original_widescreen_hacked = widescreen_hacked;
```

Comment:
Please consider remove the flag using `ReadMem32_nommu(0x0c1e7948)` or current `ScreenStretching`

Reply:
you are right